packages/compass/src/main/auto-connect.spec.ts

```diff
@@ -20,6 +20,15 @@ const URI_WITH_DISALLOWED_SERVICE_HOST_AUTH_MECHANISM_PROP =
 const URI_WITH_ALLOWED_SERVICE_HOST_AUTH_MECHANISM_PROP =
   'mongodb://localhost:27017/?authMechanism=GSSAPI&authSource=%24external&authMechanismProperties=CANONICALIZE_HOST_NAME%3Aforward';
 
+/**
+ * @securityTests Automatic Connection Establishment Tests
```

Comment:
additional _s_?

Reply:
Yup, thanks for catching!

```suggestion
 * @securityTest Automatic Connection Establishment Tests
```